### PR TITLE
add scaffolding for action

### PR DESCRIPTION
**Pull request by hamelsmu**

### CODE REVIEW

File: 00_core.ipynb [hunk 1]
```diff
@@ -207,7 +213,17 @@
...
+    "To create a markdown changelog, first create a `FastRelease` object, passing a mapping from GitHub labels to markdown titles. Put your github token in a file named `token` at the root of your repo.  `FastRelease` attempts to fetch values for arguments from the following locations if not supplied:\n",
+    "\n",
+    "- **owner:** fetched from the field `user` in `settings.ini`.  This is the owner name of the repository on GitHub. For example for the repo `fastai/fastcore` the owner would be `fastai`.\n",
+    "- **repo:** fetched from the field `lib_name` in `settings.ini`.  This is the name of the repository on GitHub.  For example for the repo `fastai/fastcore` the owner would be `fastcore`.\n",
+    "- **token:** fetched from a file named `token` at the root of your repo.  Creating a token is discussed in [the setup](https://fastrelease.fast.ai/#Set-up) section.\n",
+    "- **groups:** fetched from the field `label_groups` in `settings.ini`, which is a JSON string.  This is a mapping from label names to titles in your release notes.  For example, this is a valid specification for `label_groups` in your `settings.ini` file:\n",
```
Comment (jph00) [inline 00_core.ipynb:221]: Clarify that this is optional, and the others are required, and say what the default is.

File: 00_core.ipynb [hunk 2]
```diff
@@ -240,7 +256,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The default mapping from labels to markdown headings is:\n",
+    "Calling this method fetches relevant pull requests and issues from the GitHub API, and categorizes these entities according to a user-supplied mapping from labels to markdown headings.  The default mapping from labels to markdown headings is:\n",
```
Comment (jph00) [inline 00_core.ipynb:259]: Most docs guidelines I've seen recommend removing phrases like "calling this method...". So you could start this sentence at "Fetches"

File: settings.ini [hunk 3]
```diff
@@ -25,4 +25,6 @@ doc_baseurl = /
 git_url = https://github.com/fastai/fastrelease/tree/master/
 lib_path = fastrelease
 title = fastrelease
-
+label_groups = {"breaking": "Breaking Changes",
```
Comment (jph00) [inline settings.ini:28]: Might be clearer to leave this out, since otherwise it kinda implies like it's required here.

**Review by jph00 — CHANGES_REQUESTED** [review]
Thanks Hamel - just some minor suggestions

**Comment by hamelsmu** [comment]
@jph00 ready for review

**Pull request merged**